README.md

```diff
@@ -37,6 +37,18 @@ ReactDOM.render(
 );
 ```
 
+##### API:
+| Property      | Type          | Default             | Description |
+| ------------- |:-------------:|:------------:       | ----------- |
+| head          | node          | null                | DOM node that gets inserted before the children of the frame. Note this is injected into the body of frame (see the blog post for why). This has the benefit of being able to update and works for stylesheets.
+| initialContent | string       | `<!DOCTYPE html><html><head></head><body><div></div></body></html>`               | The initial html injected into frame. It is only injected once, but allows you to insert any html into the frame (e.g. a head tag, script tags, etc). Note that it does *not* update if you change the prop. Also at least one div is required in the body of the html, which we use to render the react dom into.
+| mountTarget          | string          | `document.body`                | CSS selector (#target/.target) that specifies where in the `initialContent` of the iframe, children will be mounted in. ```html
+<Frame
```

Comment:
This code block is broken.

Reply:
@SchneiderOr can you fix up the above and we can get this merged in.

Reply:
Yes, saddly ive dont that already but it seems that ive lost the updated file.

Ill rewrite it soon